docs/pet-kata/slides.md

```diff
@@ -909,26 +909,26 @@ public void getAgeStatisticsOfPets()
 {
   // Try to use a MutableIntList here instead
   // Hints: flatMap = flatCollect, map = collect, mapToInt = collectInt
-  MutableIntList petAges = this.people
-      .flatCollect(Person::getPets)
-      .collectInt(Pet::getAge);
+  var petAges = this.people
```

Comment:
I would leave the type in here because this is a slide and the reader will not be able to determine the type on their own.

`MutableIntList petAges = this.people`